Android/app/src/main/java/uncommon/common/activity/ReservationActivity.java

```diff
@@ -250,8 +251,9 @@ public void onClick(View view) {
                             .OnClickListener() {
                         @Override
                         public void onClick(DialogInterface dialog, int which){
+                            //TODO(gayeon):send reservation data to server
                             JSONObject requestBody = new JSONObject();
-                            requestBody.put("userEmail", "jmj@kookmin.ac.kr");
+                            requestBody.put("userEmail", "kahye5232@naver.com");
```

Comment:
Why is this sent to this email address? Shouldn't this take the user's email address?